test/uk/gov/hmrc/pensionschemereturn/connectors/PsrConnectorSpec.scala

```diff
@@ -749,17 +749,17 @@ object PsrConnectorSpec {
       |        },
       |        "borrowing": {
       |            "recordVersion": "164",
-      |            "moneyWasBorrowed": "No",
+      |            "moneyWasBorrowed": "Yes",
       |            "noOfBorrows": 1,
       |            "moneyBorrowed": [
       |                {
-      |                    "dateOfBorrow": "2022-10-18",
-      |                    "amountBorrowed": 2000,
-      |                    "schemeAssetsValue": 0,
-      |                    "interestRate": 5.55,
-      |                    "borrowingFromName": "Loans R Us",
-      |                    "connectedPartyStatus": "02",
-      |                    "reasonForBorrow": "We needed the money."
+      |                    "dateOfBorrow": "2023-10-19",
+      |                    "amountBorrowed": 1.7976931348623157E+308,
```

Comment:
Why are these in exponential format?

Reply:
`1.7976931348623157E+308` represents `Double.MaxValue` of Java.

Reply:
Yes, but should it be string formatted in ETMP? E.g. like `"100000000.00"` (or similar) . Would ETMP accept the exponential format? Maybe to check in the EPID document or schema.

Reply:
format is number in schema, not string. 

and it can understand the exponential representation. please see this scheme test with exp value.
https://github.com/hmrc/pension-scheme-return/blob/53840535f61cb10be0fa6a24a69b2e7dac7ca365/test/resources/epid-1444-assets-only-valid-example.json#L84

Reply:
Great! Thanks for checking.